.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
             unity-installation-path: /Applications/Unity/
             unity-root: /Applications/Unity/Hub/Editor/
             unity-path: Unity.app/Contents/MacOS
-          - unity-version: 2019.4.21f1
+          - unity-version: 2019.4.30f1
```

Comment:
Do we really need this here if we have it in the matrix?

Reply:
We intend to run CI with multiple Unity versions: 2019, 2020, 2021, and I put it there so I can associate the version changeset with the version number for installing via the hub.
Like .NET does it [here](https://github.com/getsentry/sentry-dotnet/blob/465e9031c24d2574ded6c5f6e686eb9030c6e879/.github/workflows/build.yml#L19).